client-v2/src/components/FeedSectionHeader.tsx

```diff
@@ -170,14 +170,16 @@ class FeedSectionHeader extends Component<ComponentProps> {
           <FadeTransition active={!isCurrentFrontsMenuOpen} direction="right">
             {editMode === 'editions' ? (
               editionsIssue ? (
-                <EditionIssueInfo>
-                  <EditionTitle>
-                    {startCase(editionsIssue.displayName)}
-                  </EditionTitle>
-                  <EditionDate>
-                    {new Date(editionsIssue.issueDate).toDateString()}
-                  </EditionDate>
-                </EditionIssueInfo>
+                <Link to="/v2/manage-editions/daily-edition">
```

Comment:
I suspect we'd want this to link to the specific edition rather than the Daily? Doesn't look like this is on the EditionsIssue type though - https://github.com/guardian/facia-tool/blob/master/client-v2/src/types/Edition.ts#L24-L35 so we'd need to add it.

Reply:
The idea was just to be able to navigate back to the date picker from within the tool. It's a bit unnatural to have to go to the bookmark.

Reply:
👍 I presume we don't always want to go back to daily-edition though, as we'll have multiple additions?

Let's ship this though and update later if needed.

Reply:
Ah - I understand your point now. But agree.